helm/templates/crd.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This file too is a duplicate of an existing CRD YAML and must not be copied. This will surely cause problems in the near future.